Sources/GRPCCodeGen/layers/CodeGenerationRequest.swift

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2023, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+/// Describes the services, dependencies and trivia from an IDL file,
+/// and the IDL itself through its specific serializer and deserializer.
+struct CodeGenerationRequest {
+  /// The name of the source file containing the IDL.
+  var fileName: String
+
+  /// The swift imports that this file depends on.
```

Comment:
```suggestion
  /// The Swift imports that the generated file should depends on.
```

Reply:
You could maybe also use `- SeeAlso:` to link to `Dependency`?